spring-boot/fdd3f12ee0f92ac18844c08bf71df39feebb6673/original/README.md

```diff
@@ -1,15 +1,15 @@
  ## 1. Clone the project 
     git clone https://github.com/spring-projects/spring-boot.git
 
-## 2. Checkout to merge the commit hash
+## 2. Open the project and do checkout to merge the commit hash
     git checkout -f fdd3f12ee0f92ac18844c08bf71df39feebb6673
 
 ## 3. Open the parent directory (spring-boot/spring-boot-parent/pom.xml) and change all the urls:
 ```xml
     http://repo.spring.io change to https://repo.spring.io
 ```
 
-## 4. Open the pom file in **spring-boot/spring-boot-autoconfigure/pom.xml** and add the code:
+## 4. Open the pom file in **spring-boot/spring-boot-autoconfigure/pom.xml** and add the code before the tag  </project>:

```

Comment:
Segui as novas modificações, e acontecem alguns problemas relacionados a versão de algumas dependências.
Você modificou/adicionou algo sobre isso?

Exemplo de erro:
`[ERROR] 'dependencies.dependency.version' for org.springframework:spring-jdbc:jar is missing. @ org.springframework.boot:spring-boot-autoconfigure:[unknown-version], /home/[...]/spring-boot/spring-boot-autoconfigure/pom.xml, line 233, column 15`

Reply:
Não modifiquei nada, segui apenas os caminhos informados. Talvez não esteja aparecendo corretamente, vou remover os **

Reply:
Resolvido.